src/web/web-service/frontend/res/js/app.js

```diff
@@ -372,5 +381,404 @@
                 ev.target.disabled = false;
             });
         };
+
+
+        $scope.ipAddr = "0.0.0.0:8080";
+        // $scope.ipaddr = location.host;
+
+        //basic information line
+        $scope.networksInfo = "Unknown";
```

Comment:
prefer single quote for string in js.
```suggestion
        $scope.networksInfo = 'Unknown';
```